commands/init.js

```diff
@@ -80,7 +82,7 @@ function createDockerCompose(dir) {
   // add a 'shell' process for persistent changes, one-off tasks
   processes.shell = _.extend(_.cloneDeep(processes.web), {
     command: 'bash',
-    volumes: ['.:/app/user']
+    volumes: ['.:' + path.join('/app/user',appDir)]
```

Comment:
There could be, but for Go it basically ends up being `/app/src/<package name>` (ex package name: github.com/heroku/go-getting-started) or `/app/user/src/<package name>`. For now I think we should stick with appending to `/app/user`. If that becomes problematic we can take this further.